tests/test_phase_psf.py

```diff
@@ -1265,8 +1270,83 @@ def test_shared_memory():
             initargs=galsim.phase_screens.initWorkerArgs()
         )
 
+@timer
+def test_pickle():
+    # This is response to ImSim issue #234
+    # https://github.com/LSSTDESC/imSim/issues/234
+
+    import multiprocessing
```

Comment:
This line is unnecessary now.  (I removed all the bits that had used it, and now you import it as mp below.)